Makefile

```diff
@@ -54,4 +54,9 @@ rpc/rpc.pb.go: rpc/rpc.proto
 stats/stats.pb.go: stats/stats.proto
 	protoc --go_out=. $^
 
-.PHONY: build test phaul-test test-bin clean lint
+vendor:
+	GO111MODULE=on go mod tidy
+	GO111MODULE=on go mod vendor
+	GO111MODULE=on go mod verify
```

Comment:
Perhaps we should use `$(GO)` instead of `go`?

Reply:
done